.env.example

```diff
@@ -1,3 +1,4 @@
 ETH_PRIVATE_KEY=
+ETH_PRIVATE_KEY1=
```

Comment:
Why is it recommended to use 2 different keys?

Reply:
Thanks for the link, but I still don't understand why do they recommend to be different. I don't see any security issue if using the same key. I am not strongly against adding it in the example, but it would be good to also put a note why a different key is used for the simulation and the actual request.

Reply:
I would also consider better naming convention for the environment variables:

```
FLASHBOT_AUTH_PRIVATE_KEY
FLASHBOT_SIGN_PRIVATE_KEY
```

Reply:
yes this is a good reason, lets add a note for that explanation and yes lets rename the keys to your suggestion.